### PR TITLE
Intro conclusion popups

### PR DESCRIPTION
**Pull request by drbalar**
Demo: https://www.loom.com/share/f89205e16c1940e7a9ad2f2947be257b

### CODE REVIEW

File: scripts/mockdata/scenarios.json [hunk 1]
```diff
@@ -54,7 +74,7 @@
         "data": [
           { "text": "just text, no images" },
           {
-            "text": "ANOTHA ONE",
+            "text": "Conclusion: ANOTHA ONE",
```
Comment (ahmedhamodi) [inline scripts/mockdata/scenarios.json:77]: Perhaps I'm missing something, but in the video it shows `Conclusion: Just text no image` but here it says `Conclusion: ANOTHA ONE`. Where is the text being rendered from?

**Review by ahmedhamodi — COMMENTED** [review]
Some lint errors. Otherwise, video looks good from an end user perspective. A small comment on where the actual text is being pulled from, appears to be mismatched?

File: app/public/templates/scene.html [hunk 2]
```diff
@@ -123,37 +127,47 @@
...
+
+          addTransition(transitionData);
+        });
+
+        function addTransition(transition_data) {
+          let componentDataParsed = {
+            "componentType": "text-pane",
+            "jsonData": transition_data
+          };
+          console.log(componentDataParsed);
```
Comment (jaydhulia) [inline app/public/templates/scene.html:168]: `console.log` :D

File: scripts/mockdata/scenarios.json (hunk 1 above)
Comment (drbalar) [reply scripts/mockdata/scenarios.json:77]: oh lol the video was from before I did merge with vivian's latest changes

Comment (drbalar) [reply scripts/mockdata/scenarios.json:77]: great observation 🧐

Comment (ahmedhamodi) [reply scripts/mockdata/scenarios.json:77]: Ah okay, as long as the functionality is the same then I'm good with that :)

**Review by jaydhulia — APPROVED** [review]

**Pull request merged**